.github/workflows/tests.yml

```diff
@@ -82,3 +89,15 @@ jobs:
       - name: Run tests
         run: |
           ./run-tests.sh
+
+      - name: Install deps for frontend tests
+        working-directory: ./invenio_administration/assets/semantic-ui/js/invenio_administration
```

Comment:
is this directory correct (`invenio_administration`)?  Same question applies to the tasks below